inc/sainexthopgroup.h

```diff
@@ -184,6 +184,19 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Hierarchical next hop group.
+     *
+     * false: Control plane will set this flag as false when Nexthop group consists of only IP nexthop
+     * true: Control plane will set this flag as true when nexthop group consists of both tunnel and IP nexthop
+     *
+     * @type bool
```

Comment:
I suspect this will need to be converted to an integer value someday. 3-level LB does exist.

Reply:
Agree, This we can address later as needed